packages/react-components/react-toast/src/components/Toast/Toast.types.ts

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { Announce } from '../AriaLive/AriaLive.types';
+import { Toast as VanillaToast } from '../../state';
```

Comment:
This type collision does not seem like something good. Did you consider renaming it? ToastItem?

Reply:
yeah this should be addressed, but should be done in a PR scoped to this https://github.com/microsoft/fluentui/issues/28108